source/crossover/crossover.py

```diff
@@ -0,0 +1,9 @@
+from typing import List
+from source.population.individual import Individual
+from abc import ABC, abstractmethod
+
+class Crossover(ABC):
```

Comment:
Co ty na to żeby crossover_number i crossover_probability dać do konstruktora bazowego, plus zamiast crossover_number dać bezpośrednio crossover_size (jako ilość zwracanych individuals'ów)?

Reply:
W sumie to moje crossover_number pokrywa się z crossover_size. Zmieniłam nazwę na crossover_size. Przerzuciłam  crossover_probability do konstruktora bazowego. Crossover_number nie będzie chyba wszędzie używane, więc go nie dodałam

Reply:
Gdzie dokładnie crossover_number/size nie będzie używane?